### PR TITLE
docs: update delegate example to include ipfs setup instructions

### PR DESCRIPTION
**Pull request by jacobheun**
This updates the delegate routing example to include instructions for setting up the go-ipfs node locally to support the needed API endpoints.

I also added an "Add Data" field to the UI and instructions to demonstrate content providing being delegable.

### CODE REVIEW

File: examples/delegated-routing/README.md [hunk 1]
```diff
@@ -11,17 +11,17 @@ Once you've completed the example, you should try enabled the DHT and see what k
 various Peer Discovery modules and see the impact it has on your Peer count.
 
 ## Prerequisite
```
Comment (dirkmc) [inline examples/delegated-routing/README.md]:
```suggestion
```

File: examples/delegated-routing/README.md [hunk 2]
```diff
@@ -11,17 +11,17 @@ Once you've completed the example, you should try enabled the DHT and see what k
...
 ## Running this example
 
-1. Install IPFS locally if you dont already have it. [Install Guide](https://docs.ipfs.io/introduction/install/)
-2. Run the IPFS daemon: `ipfs daemon`
-3. The daemon will output a line about its API address, like `API server listening on /ip4/127.0.0.1/tcp/8080`
-4. In another window output the addresses of the node: `ipfs id`. Make note of the websocket address, it will contain `/ws/` in the address.
-5. In `./src/libp2p-bundle.js` check if the host and port of your node are correct, according to the previous step. If they are different, replace them.
-6. In `./src/App.js` replace `BootstrapNode` with your nodes Websocket address from step 4.
-7. Start this example:
+1. Install IPFS locally if you dont already have it. [Install Guide](https://docs.ipfs.io/introduction/install/).
```
Comment (dirkmc) [inline examples/delegated-routing/README.md]:
```suggestion
1. Install go-ipfs locally if you dont already have it. [Install Guide](https://docs.ipfs.io/introduction/install/).
```

**Comment by lidel** [comment]
I tried to follow instructions but I am getting error 404 when calling `dht/findpeer` with latest go-ipfs master as noted in https://github.com/ipfs/js-ipfs/issues/2155#issuecomment-505206651:



Any Idea what I am missing?

**Comment by jacobheun** [comment]
@lidel I started with a fresh setup locally and I am hitting issues now too. Going to look more into this, I'm assuming it's a configuration issue because my ipfs version has not changed.

**Comment by alanshaw** [comment]
I'm planning on giving this a wirl tomorrow - will let you know how it works out!

**Comment by olizilla** [comment]
@alanshaw but tomorrow never came.

**Comment by alanshaw** [comment]
😭 sorry! https://www.youtube.com/watch?v=JuntWQBJ5GM

**Comment by autonome** [comment]
I'll never forget the look in Oli's eyes when I brought up that movie in the course of regular conversation.

**Comment by lidel** [comment]
Yikes, we forgot about this one :see_no_evil: 
I filled https://github.com/ipfs/go-ipfs/issues/8059 and added it to our maintenance backlog.

**Comment by vasco-santos** [comment]
@lidel this is quite outdated. We should close this as https://github.com/libp2p/js-libp2p/pull/507#issuecomment-655478849 is the new PR, which is also blocked

**Comment by lidel** [comment]
K,  let's continue in https://github.com/libp2p/js-libp2p/pull/507#issuecomment-655478849

**Pull request closed without merging**